### ISSUE TITLE
1.1.5 NoPorts  Design Nits

### ISSUE THREAD

**Issue by Dariamarg** [atsign-foundation/noports#1614]
### Is your feature request related to a problem? Please describe.

Here are some suggestions, if time allows!

### Describe the solution you'd like

**Dashboard**
- [x] Can we move the settings icon to match the padding to the left (where the logo is)
![Image](https://github.com/user-attachments/assets/1e104e4a-cc1a-4aa8-93f9-44ba9e6c984d)

**Settings**
- [x] Can we get make the hover of the toggle black (not purple) #25252525
![Image](https://github.com/user-attachments/assets/487b7b6b-7604-486b-97b2-f7e98758ad9b)

- [x] Can we get the checkbox to be on the left side, in order to match
 the one above "Overide all profiles...."
![Image](https://github.com/user-attachments/assets/229bbf6c-e963-4118-98e8-b706be69816d)

- [ ] Replace the advanced/ simple screens with ones that match the In-app UI (@Dariamarg)
![Image](https://github.com/user-attachments/assets/bd9e6693-4bde-4268-a72b-bc44c9e36124)

**Edit/ Create a Profile**

- [x] Can the submit button be suppressed until all fields are entered? <img width="980" alt="Image" src="https://github.com/user-attachments/assets/7605aecc-cf31-4717-87f8-aa56a90561ec" />

- [x] For custom relays can we add some text in the box along the lines of 'Custom' and add a radio dial to show that it is a selection (for example) <img width="694" alt="Image" src="https://github.com/user-attachments/assets/e3290aa7-fac9-4bf2-a72e-f190f4d39807" />

- [x] Can we also have the fields be fixed closer to each other (for example in fullscreen 'Device atSign' and 'Device Name' have a large gap); just have them be side by side <img width="1462" alt="Image" src="https://github.com/user-attachments/assets/4b468558-70da-4996-a3f1-8525cbec5e17" />

**Comment by Dariamarg** [comment]
Here are the advanced and simple view screenshots
[NoPorts Marketing Material (Dashboard).zip](https://github.com/user-attachments/files/18102584/NoPorts.Marketing.Material.Dashboard.zip)

**Comment by CurtlyCritchlow** [comment]
@Dariamarg is this what you have in mind?

![Image](https://github.com/user-attachments/assets/8c75b948-b2db-4ea6-94e6-0af4ab4fb8ff)

**Comment by CurtlyCritchlow** [comment]
@Dariamarg Can the submit button be suppressed until all fields are entered? Currently if the use click submit it will trigger a validation error on all the fields that have errors.

**Comment by CurtlyCritchlow** [comment]
@Dariamarg this is how the UI looks on a large screen now, It looks the same on smaller screen, I don't like this UI on a large screen, what do you recommend?

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/db573fb1-7147-44a0-a46a-82b9f497be6b" />

**Comment by CurtlyCritchlow** [comment]
Her is another view with it centered. I'll have to refactor the app bar so I aligns in the centre. I'll do so if centering the content on a large screen is in keeping with your recommendation.
![Image](https://github.com/user-attachments/assets/2d64a434-2646-4c4e-8a2b-1c6603be069b)

**Comment by Dariamarg** [comment]
I think centering is a great idea! Maybe have it at full width so that theres no scroll (like in the relay section)

**Comment by CurtlyCritchlow** [comment]
> [@Dariamarg](https://github.com/Dariamarg) is this what you have in mind?
> 
> ![Image](https://github.com/user-attachments/assets/8c75b948-b2db-4ea6-94e6-0af4ab4fb8ff)

@Dariamarg Your thoughts on this.

**Comment by Dariamarg** [comment]
> > [@Dariamarg](https://github.com/Dariamarg) is this what you have in mind?
> > ![Image](https://private-user-images.githubusercontent.com/44856451/397385221-8c75b948-b2db-4ea6-94e6-0af4ab4fb8ff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ2MjU1ODcsIm5iZiI6MTczNDYyNTI4NywicGF0aCI6Ii80NDg1NjQ1MS8zOTczODUyMjEtOGM3NWI5NDgtYjJkYi00ZWE2LTk0ZTYtMGFmNGFiNGZiOGZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE5VDE2MjEyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThhNzJhYTMwYmQ5NmI2MWI3ZjFlY2Y3ZmM0ZjdmYjY4NzU5OGZhMWVlMmI2NzI3YmZkYzQxOTE4YjlhMzQwOGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.otWAsf3RUlttclpJ8etVZooMY6YYY_e_OHfmv-DkN-U)
> 
> @Dariamarg Your thoughts on this.

Yeah just on hover, but white when selected (essentially replace purple -> black

**Comment by CurtlyCritchlow** [comment]
Replace the advanced/ simple screens with ones that match the In-app UI Will be completed in the next release of the app. Given that the UI will be updated to accommodate the testing profile.
